stable/envoy/templates/servicemonitor.yaml

```diff
@@ -27,4 +27,12 @@ spec:
     matchLabels:
       app: {{ template "envoy.name" . }}
       release: {{ .Release.Name }}
-{{- end }} 
+  {{- with .Values.serviceMonitor.targetLabels }}
+  targetLabels:
+{{ toYaml . | trim | indent 4 -}}
```

Comment:
I think this can be even better if we use `nindent` like:

```
  targetLabels:
    {{ toYaml . | trim | nindent 4 -}}
```

Please try if you have any chance, and I'd appreciate it if you could submit a follow-up PR for this chart as well 😄